docs/command-reference/tanzu_apps_workload_apply.md

```diff
@@ -63,6 +63,7 @@ tanzu apps workload apply --file workload.yaml
       --tail                           show logs while waiting for workload to become ready
       --tail-timestamp                 show logs and add timestamp to each log line while waiting for workload to become ready
   -t, --type type                      distinguish workload type
+      --update-strategy string         specify configuration file update strategy (supported strategies: merge, replace) (default "merge")
```

Comment:
I'd recommend a slightly shorter version (adjust spacing as needed - I'm pretty sure I got it wrong):

--update-strategy   string          specify configuration file update strategy (supported strategies: merge[default], replace )

Reply:
This is the way cobra is adding the help text

Reply:
Assuming that means we can't do what I suggested so I'll resolve this conversation.